docs/source/coder/globalKeys.rst

```diff
@@ -0,0 +1,281 @@
+Global Event Keys
+=================
+
+Global event keys are single keys (or combinations of a single key and one or
+more "modifier" keys such as Ctrl, Alt, etc.) with an associated Python
+callback function. This function will be executed if the key (or
+key/modifiers combination) was pressed.
+
+.. note::
+
+   Global event keys only work with the `pyglet` backend, which is the default.
+
+PsychoPy fully automatically monitors and processes key presses during most
+portions of the experimental run, for example during
+`core.wait()` periods, or when calling `win.flip()`. If a global
+event key press is detected, the specified function will be run
+immediately. You are not required to manually poll and check for key
+presses. This can be particularly useful to implement a global
+"shutdown" key, or to trigger laboratory equipment on a key press
+when testing your experimental script -- without cluttering the code.
+But of course the application is not limited to these two scenarios.
+In fact, you can associate any Python function with a global event key.
+
+All active global event keys are stored in `event.globalKeys`.
+
+Adding a global event key (simple)
+----------------------------------
+First, let's ensure no global event keys are currently set by calling
+func:`event.globalKeys.clear`.
+::
+    >>> from psychopy import event
+    >>> event.globalKeys.clear()
+
+To add a new global event key, you need to invoke
+func:`event.globalKeys.add`. This function has two required arguments: the
+key name, and the function to associate with that key.
+::
+    >>> key = 'a'
+    >>> def myfunc():
+    ...     pass
+    ...
+    >>> event.globalKeys.add(key=key, func=myfunc)
+
+Look at `event.globalKeys`, we can see that the global event key has indeed
+been created.
+::
+
+    >>> event.globalKeys
+    <_GlobalEventKeys :
+        [A] -> 'myfunc' <function func at 0x10669ba28>
```

Comment:
Hah, you caught me there. I edited those lines after copy & pasting from the IPython console, and forgot this spot. :)